manifests/istio-control/istio-discovery/templates/telemetryv2_1.4.yaml

```diff
@@ -1,3 +1,5 @@
+{{- if eq .Values.revision "" }}
```

Comment:
You don’t need the complex clause in the name now.
Here and below.

Is it possible that 1.5 proxy still talks to 1.6 pilot? What prevents it?

Reply:
In theory, you can set the discovery address on the 1.5 proxy to point to the new revision - we don't prevent that in any way. In practice this should never happen though - the old proxies will be referencing just `istiod` or `istio-pilot`, not the new `istiod-REVISION` service.